coresdk/src/coresdk/types.h

```diff
@@ -43,6 +43,14 @@ enum font_style
     UNDERLINE_FONT   = 4
 };
 
+/// Provides an overload that allows combining font styles with the bit or 
+/// operator.
+/// Source: http://stackoverflow.com/a/1448478
+inline font_style operator|(font_style a, font_style b)
```

Comment:
Yeah I agree - I wasn't sure but I figured I'd see what you guys thought. I did this because this is how SDL handles font styles. We can add some methods to the backend to emulate this and maintain the desired flags ourselves potentially?